src/api/socket.js

```diff
@@ -86,6 +86,7 @@ export const createDeviceSocket = (
                 o.next({
                   type: "device-permission-requested",
                   wording: "Ledger Manager",
+                  forManager: true,
```

Comment:
ok i don't understand why this is needed because I'm not seeing its usage in any place.

Reply:
dead code from the first version indeed, good catch